account_invoice_fixed_discount/models/account_move.py

```diff
@@ -5,6 +5,23 @@
 from odoo.exceptions import ValidationError
 
 
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _recompute_tax_lines(self, recompute_tax_base_amount=False):
+        vals = {}
+        for line in self.invoice_line_ids.filtered("discount_fixed"):
+            vals[line] = {"price_unit": line.price_unit}
+            price_unit = line.price_unit - line.discount_fixed
+            line.update({"price_unit": price_unit})
+        res = super(AccountMove, self)._recompute_tax_lines(
+            recompute_tax_base_amount=False
```

Comment:
Super function does not have this parameter as argument, thus the error. You need to remove it from here

Reply:
I Think you need to fetch code from V13 because it's need.
![Selection_040](https://user-images.githubusercontent.com/52144935/72243639-05f43d00-361f-11ea-8e4f-0793205300cb.png)

Reply:
You're right, I didn't have my Odoo code up to date. My bad

Reply:
It's fine, Thank for your review. XD